Gemfile

```diff
@@ -0,0 +1,35 @@
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "~> 4.3.3"
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem "minima", "~> 2.5"
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
+# uncomment the line below. To upgrade, run `bundle update github-pages`.
+# gem "github-pages", group: :jekyll_plugins
+# If you have any plugins, put them here!
+group :jekyll_plugins do
+  gem "jekyll-feed", "~> 0.12"
+end
+
+# Windows and JRuby does not include zoneinfo files, so bundle the tzinfo-data gem
+# and associated library.
+platforms :mingw, :x64_mingw, :mswin, :jruby do
+  gem "tzinfo", ">= 1", "< 3"
+  gem "tzinfo-data"
+end
```

Comment:
Are we printing timezones anywhere?

Reply:
This is boilerplate inserted into the Gemfile when jekyll creates a new project from the CLI. I've never tested whether it _actually_ is needed to run, but it also doesn't really matter. It's extra work to remove it, but there's no real cost to leaving it.

Reply:
As mentioned below, I disagree with you here. Less code is better and from a security POV there *is* a cost to pay (and I don't expect us to spend lots and lots of time maintaining this, so there's a risk it will be running unattended for long-ish stretches of time).

Reply:
I agree philosophically. One thing worth noting, though, is that this will likely run only one time -- when this is deployed. The only other runs would be if we need to change the redirects, but that may never happen.